Userland/Services/WindowServer/WMClientConnection.cpp

```diff
@@ -171,4 +171,4 @@ void WMClientConnection::handle(const Messages::WindowManagerServer::SetWindowTa
     window.set_taskbar_rect(message.rect());
 }
 
-}
+}
```

Comment:
This is now removing a newline at the end of the file (which will fail the build soon..)

Reply:
Yeah... :sweat_smile: just fixed that, sorry